<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLeftArea"/>
    <addaction name="actionRightArea"/>
    <addaction name="actionTopArea"/>
    <addaction name="actionBottomArea"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLeftArea">
   <property name="text">
    <string>Left Area</string>
   </property>
  </action>
  <action name="actionRightArea">
   <property name="text">
    <string>Right Area</string>
   </property>
  </action>
  <action name="actionTopArea">
   <property name="text">
    <string>Top Area</string>
   </property>
  </action>
  <action name="actionBottomArea">
   <property name="text">
    <string>Bottom Area</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
